star-ros2/src/star_gateway_bridge/include/star_gateway_bridge/star_gateway_bridge_node.hpp

```
@@ -8,8 +8,8 @@
 // STAR Project - Texas A&M University
 // January 2026
 
-#ifndef STAR_GATEWAY_BRIDGE_INCLUDE_STAR_GATEWAY_BRIDGE_STAR_GATEWAY_BRIDGE_NODE_HPP_
-#define STAR_GATEWAY_BRIDGE_INCLUDE_STAR_GATEWAY_BRIDGE_STAR_GATEWAY_BRIDGE_NODE_HPP_
+#ifndef STAR_GATEWAY_BRIDGE__STAR_GATEWAY_BRIDGE_NODE_HPP_
+#define STAR_GATEWAY_BRIDGE__STAR_GATEWAY_BRIDGE_NODE_HPP_
```

Clang (14.0.6)

[warning] declaration uses identifier 'STAR_GATEWAY_BRIDGE__STAR_GATEWAY_BRIDGE_NODE_HPP_', which is a reserved identifier

(bugprone-reserved-identifier)